File: output/qwen_2.5_72b_instruct/avatar/algo_based_translation/java/python/codeforces_99_A.py

from sys import stdin
from decimal import Decimal

def main():
    str_input = stdin.readline().strip()
    str = list(str_input)
    dot_index = 0
    for i in range(len(str)):
        if str[i] == '.':
            dot_index = i
            break
    if str[dot_index - 1] == '9':
        print("GOTO Vasilisa.")
    elif int(str[dot_index + 1]) > 4:
        _str = ''.join(str[:dot_index])
        result = str(int(_str) + 1)
        print(result)
    else:
        print(''.join(str[:dot_index]))

if __name__ == "__main__":
    main()

File: output/qwen_2.5_72b_instruct/avatar/algo_based_translation/python/java/codeforces_99_A.java

import java.util.Scanner;
import java.util.List;
import java.util.Arrays;

public class codeforces_99_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String inputString = scanner.nextLine();
        processInput(inputString);
        scanner.close();
    }

    public static void processInput(String inputString) {
        List<String> l = Arrays.asList(inputString.split("\\."));
        List<String> s = Arrays.asList(l.get(0).split(""));
        List<String> p = Arrays.asList(l.get(1).split(""));
        int i = Integer.parseInt(p.get(0));

        if (s.get(s.size() - 1).equals("9")) {
            System.out.println("GOTO Vasilisa.");
        } else if (!s.get(s.size() - 1).equals("9") && i < 5) {
            String result = String.join("", s);
            System.out.println(result);
        } else {
            String result = String.join("", s);
            int incrementedResult = Integer.parseInt(result) + 1;
            System.out.println(Integer.toString(incrementedResult));
        }
    }
}

